sdk/servicebus/azure-messaging-servicebus-track1-perf/src/main/java/com/microsoft/azure/servicebus/perf/MessageUtil.java

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.microsoft.azure.servicebus.perf;
+
+/**
+ * A utility class for message.
+ */
+public class MessageUtil {
+
+    /**
+     * Generate the message content with given {@code messageSize} value.
+     *
+     * @param messageSize of the message to be generated.
+     * @return the generated message.
+     */
+    public static String generateMessageContent(int messageSize) {
```

Comment:
this can be replaced with 

```java
    public static String generateRandomString(int targetLength) {
        int leftLimit = 97;
        int rightLimit = 122;
        Random random = new Random();

        String generatedString = random.ints(leftLimit, rightLimit + 1)
            .limit(targetLength)
            .collect(StringBuilder::new, StringBuilder::appendCodePoint, StringBuilder::append)
            .toString();
        return generatedString;
    }
```

Reply:
Also move this to TestDataCreationHelper class in the perf framework.